src/base/lib/Convex/Class.hs

```diff
@@ -372,6 +378,20 @@ instance Monad m => MonadError e (MonadBlockchainCardanoNodeT e m) where
 runMonadBlockchainCardanoNodeT :: LocalNodeConnectInfo -> MonadBlockchainCardanoNodeT e m a -> m (Either (MonadBlockchainError e) a)
 runMonadBlockchainCardanoNodeT info (MonadBlockchainCardanoNodeT action) = runExceptT (runReaderT action info)
 
+{-| Query the node for the current era.
+Fails with 'UnsupportedEra' if the current era is not one of (Babbage, Conway)
+-}
+getSupportedEra :: forall e m.
```

Comment:
A more representative name might be `getCurrentNodeEra` or just `getCurrentEra` or `getCurrentSupportedEra`?